Automattic/sensei
# Prevent early translation loading

Resolves #7696

## Proposed Changes

There is a new change in WordPress 6.7 that prevents plugins from loading translations too early, without waiting for more parts of WordPress to be loaded.

This PR fixes all translations that are loaded early while preserving the code logic.

## Testing Instructions


1. Use WordPress 6.7.
2. Set site language to something other than en_US.
3. Ensure all translations are installed.
4. Make sure there are no PHP translation warnings similar to this: `PHP Notice:  Function _load_textdomain_just_in_time was called incorrectly`
5. Test the plugin and make sure everything works as before.

## Pre-Merge Checklist
